www/includes/easyparliament/templates/html/mp/profile.php

```diff
@@ -74,13 +74,13 @@
 
                     </div>
                     <?php } ?>
-                <?php } else if (count($policyPositions->positions) > 0 || count($sorted_diffs) > 0): ?>
+                <?php } else if ($party_member_count > 0 && (count($policyPositions->positions) > 0 || count($sorted_diffs) > 0) ): ?>
```

Comment:
Is it possible for this to be 0 if it was the member's own party that was used for the count?